workflow-templates/im-test-cypress-schedule.yml

```diff
@@ -0,0 +1,130 @@
+#    - Testing projects that use Cypress.
+#
+# TODO Prerequisites:
+#    - Add secret APPLICATIONINSIGHTS_CONNECTION_STRING to every environment you are testing.
+#    - Update your cypress test config to include the im.exporttoappinsights package. Instructions found here: https://github.com/im-practices/techhub/blob/main/docs/Azure/content/Health-Checks/Cypress-UI-Health-Check.md
+#    - If the project contains a local nuget.config remove it.  It interferes with restoring packages on the GitHub Action runners.
+#    - In order to use this workflow a few changes need to be made to the cypress test configuration:
+#      1. Make the following npm package changes in your cypress test directory
+#         npm uninstall mocha-teamcity-reporter
+#         npm install --save-dev mochawesome mochawesome-merge mochawesome-report-generator
+#      2. Modify the cypress run section of the ci script by replacing the '--reporter ./mocha-teamcity-reporter' with '--reporter mochawesome --reporter-options overwrite=false,html=false,json=true'. For instance:
+#         old: "cy:ci" : "....&& cypress run --project ./ --reporter ./mocha-teamcity-reporter \"",
+#         new: "cy:ci" : "....&& cypress run --project ./ --reporter mochawesome --reporter-options overwrite=false,html=false,json=true \"",
+#      3. Add the following pre-ci script.  Your name will vary based on your cypress ci script name.  This removes results between test runs.
+#         "precy:ci": "rm -rf mochawesome-report/",
+#      4. Add the following script that merges all test json files and generates an html report.  Your name may vary based on your project conventions.  If the output file or results dir are changed, update the env values in the workflow.
+#         "cy:report": "mochawesome-merge mochawesome-report/*.json > merged-results.json && marge --reportDir cypress-results merged-results.json",
+#                                                                                             ^^ marge is not a typo, it's a command for the "mochawesome report generator"
+
+name: 🧪 Run Cypress on Schedule
+run-name: 🧪 Run Cypress on Schedule
+
+# TODO: Verify Triggers
+on:
+  schedule: # TODO: Update the schedule to run at the desired frequency. Do not run more than every 30 minutes as to not produce undue load on the system.
+    - cron: '59 * * * *' # Run Every hour # TODO: Update the frequency of the schedule.  Do not run more than every 30 minutes as to not produce undue load on the system.
+    # - cron: '1/30 * * * *' # Run Every 30 minutes # TODO: Uncomment this line if you want to run every 30 minutes.
+    # - cron: '1/40 * * * *' # Run Every 40 minutes # TODO: Uncomment this line if you want to run every 40 minutes.
+
+env:
+  GITHUB_REF: 'main' # Default branch to run against.
+  READ_PKG_TOKEN: ${{ secrets.READ_PKG_TOKEN }} # This is an org-level secret
+  JS_TEST_DIR: '' # TODO: Set to the directory containing the js tests like tests/cypress.
```

Comment:
Shouldn't this be renamed to CYPRESS_TEST_DIR?

Reply:
Updated to CYPRESS_TEST_DIR